pkg/controller/stacks/install/installjob.go

```diff
@@ -302,3 +377,13 @@ func isStackObject(obj *unstructured.Unstructured) bool {
 	return gvk.Group == v1alpha1.Group && gvk.Version == v1alpha1.Version &&
 		strings.EqualFold(gvk.Kind, v1alpha1.StackKind)
 }
+
+func isCRDObject(obj runtime.Object) bool {
```

Comment:
Should this be a shared utility function somewhere? Seems like it might be useful in other places

Reply:
Perhaps, once we have more uses.  We've been moving away from generic `utils` packages: https://github.com/crossplaneio/crossplane-runtime/issues/1